src/puppetlabs/puppetdb/http/index.clj

```diff
@@ -0,0 +1,18 @@
+(ns puppetlabs.puppetdb.http.index
+  (:require [net.cgrand.moustache :refer [app]]
+            [puppetlabs.puppetdb.http.query :as http-q]
+            [puppetlabs.puppetdb.middleware :refer [wrap-with-paging-options]]
+            [puppetlabs.puppetdb.query.paging :as paging]))
+
+(defn routes
+  [version optional-handlers]
+  (let [param-spec {:optional paging/query-params
+                    :required ["query"]}]
+    (app
```

Comment:
All the other query http endpoints use this methodology, this feels like a refactor wishlist for later.